### PR TITLE
handshake instruction 

### PR DESCRIPTION
**Pull request by jacozsz**
hi i just add simple method to execute handshake instruction for test the fingerprint sensor
i test and work in FPM10A sensor and i want to share it

added handshake instruction code, code error and the method that return true if handshake success ok

### CODE REVIEW

File: src/files/pyfingerprint/pyfingerprint.py [hunk 1]
```diff
@@ -422,6 +425,37 @@ def setPassword(self, newPassword):
         else:
             raise Exception('Unknown error '+ hex(receivedPacketPayload[0]))
 
+    def handShake(self):
+        """
+        handshake the fingerprint sensor.
```
Comment (bastianraschke) [inline src/files/pyfingerprint/pyfingerprint.py]: Please document the information you provided in the PR also here in method documentation:
* For which sensor it is
* What is the purpose of the call
* What means the result

**Review by bastianraschke — CHANGES_REQUESTED** [review]
Thanks for your PR, please check and adjust my review comment, so i can merge it.

File: .vs/VSWorkspaceState.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (bastianraschke) [inline .vs/VSWorkspaceState.json:1]: Please revert this file :)

File: src/files/pyfingerprint/pyfingerprint.py [hunk 3]
```diff
@@ -422,6 +425,44 @@ def setPassword(self, newPassword):
         else:
             raise Exception('Unknown error '+ hex(receivedPacketPayload[0]))
 
+    def handShake(self):
```
Comment (bastianraschke) [inline src/files/pyfingerprint/pyfingerprint.py:427]: I would like to name the method `testHandShake()`. What do you mean @philippmeisberger ?

**Review by bastianraschke — CHANGES_REQUESTED** [review]

File: src/files/pyfingerprint/pyfingerprint.py (hunk 3 above)
Comment (philippmeisberger) [reply src/files/pyfingerprint/pyfingerprint.py:427]: I would prefer `handshake()`. `testHandShake()` sounds like a unit-test ;)

**Comment by philippmeisberger** [comment]
@jacozsz Please exclude directory ".vs" from this PR

File: src/files/pyfingerprint/pyfingerprint.py (hunk 3 above)
Comment (bastianraschke) [reply src/files/pyfingerprint/pyfingerprint.py:427]: Okay, i agree

File: src/files/pyfingerprint/pyfingerprint.py [hunk 4]
```diff
@@ -36,6 +36,7 @@
 FINGERPRINT_GETSYSTEMPARAMETERS = 0x0F
 FINGERPRINT_TEMPLATEINDEX = 0x1F
 FINGERPRINT_TEMPLATECOUNT = 0x1D
+FINGERPRINT_HANDSHAKE = 0X17
```
Comment (bastianraschke) [inline src/files/pyfingerprint/pyfingerprint.py]: lowercase `x`

**Comment by jacozsz** [comment]
i can't exclude directory ".vs", visual studio don't let me do.

File: src/files/pyfingerprint/pyfingerprint.py (hunk 3 above)
Comment (philippmeisberger) [reply src/files/pyfingerprint/pyfingerprint.py:427]: @jacozsz Please name the method `handshake()` (all lower case) ;)

**Comment by philippmeisberger** [comment]
@jacozsz Try executing `git rm --cached .vs/*`. If files are untracked, commit the changes ;)

**Comment by bastianraschke** [comment]
Due to lack of response, the PR is closed

**Pull request closed without merging**